examples/ubuntu-ui-toolkit-gallery/po/uitk-gallery.ubports.pot

```diff
@@ -0,0 +1,966 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-02-15 09:52+0000\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: /home/lduboeuf/dev/perso/ubuntutouch/ubuntu-ui-toolkit/examples/ubuntu-ui-toolkit-gallery/About.qml:34
+#: /home/lduboeuf/dev/perso/ubuntutouch/ubuntu-ui-toolkit/examples/ubuntu-ui-toolkit-gallery/build/all/app/install/About.qml:34
```

Comment:
Not sure what happened here, but all of these shouldn't have the full path like this. Going to keep this as changes requested, as we need to solve this. I think it can wait for OTA-16 to be done though.

Reply:
oups